Mathlib/GroupTheory/Congruence.lean

```diff
@@ -246,8 +246,8 @@ variable (c)
 
 -- Quotients
 /-- Defining the quotient by a congruence relation of a type with a multiplication. -/
-@[to_additive "Defining the quotient by an additive congruence relation of a type with
-an addition."]
+@[to_additive (attr := reducible) "Defining the quotient by an additive congruence relation
```

Comment:
It's a little unclear whether this is a good idea.

Is this `Quotient` supposed to be a type synonym, or is it meant to have its own instances, separate from the ones on the global `Quotient`?

Reply:
I am not 100% on it either but I figured that parameters like `(c : Con R)` would keep instances in their own lanes.

Reply:
On the plus side, this does mean instances for `Quotient` become instances for this `Quotient` too (like `Fintype`, `Finite`, `DecidableEq`, etc.)

Reply:
Benchmarking indicates we don't have much leakage (if any).

Reply:
I think this is supposed to just be a shorthand; I don't think there is any reason to give it different instances.